staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -98,12 +99,17 @@ func NewReflector(lw ListerWatcher, expectedType interface{}, store Store, resyn
 	return NewNamedReflector(getDefaultReflectorName(internalPackages...), lw, expectedType, store, resyncPeriod)
 }
 
+// reflectorDisambiguator is used to disambiguate started reflectors.
+// initialized to an unstable value to ensure meaning isn't attributed to the suffix.
+var reflectorDisambiguator = int64(time.Now().UnixNano() % 12345)
```

Comment:
so, I'm kind of uncomfortable depending on time here.  While incredibly unlikely now, it's theoretically feasible to have this called at the same timestamp.